src/app/shared/components/Breadcrumbs/BreadcrumbsItem/BreadcrumbsItem.tsx

```diff
@@ -178,6 +177,16 @@ const BreadcrumbsItem = (props: BreadcrumbsItemProps): JSX.Element => {
     dispatch(storageThunks.downloadItemsThunk(currentFolder));
   };
 
+  const onDetailsItemButtonClicked = async () => {
+    const itemDetails: DriveItemDetails = {
+      ...currentFolder[0],
+      isShared: true,
```

Comment:
Why this `isShared` prop is always `true`?

Reply:
My mistake. It should be false for BreadcrubsItems since we don't have the dropdown in Shared View.